donate/settings/base.py

```diff
@@ -161,7 +161,8 @@ def MIDDLEWARE(self):
     FRONTEND = {
         'RELEASE_VERSION': env('HEROKU_RELEASE_VERSION'),
         'SENTRY_DSN': env('SENTRY_DSN'),
-        'SENTRY_ENVIRONMENT': env('SENTRY_ENVIRONMENT')
+        'SENTRY_ENVIRONMENT': env('SENTRY_ENVIRONMENT'),
+        'GA_TRACKING_ID': env('GA_TRACKING_ID'),
```

Comment:
Does this need to be another env var that's specified in all the heroku instances, or can we bake it into the Staging/Production django configuration, with an override for the Thunderbird app?

Reply:
it's probably easier to manage as an env var on heroku instances. This does not need to be specified on ALL instances though. If it exists, it uses it, if not, it doesn't.